doc/source/_toc.yml

```diff
@@ -85,6 +85,8 @@ parts:
       - file: data/dataset
         title: Ray Data
         sections:
+          - file: data/dataset
+            title: "What is Ray Data?"
```

Comment:
should we keep consistency across all libraries? Why this is a problem for Ray Data only?

Reply:
It's a problem for all libraries, but will let each library owner decide what they want to do. Didn't want to overload everything in a single PR.